src/libs/url_utils.js

```diff
@@ -65,10 +65,3 @@ export function buildQueryString(queriesObject) {
   const params = new URLSearchParams(removeNullEntries(queriesObject)).toString();
   return params ? `?${params}` : '';
 }
-
-export function updateQueryString(queriesObject) {
```

Comment:
Actually i liked this function :smile: 
Maybe we can use it in some places, like here https://github.com/QwantResearch/erdapfel/pull/947/files#diff-25b5c163df2044a30533a391fcc5c83ca9bdb174a2e1efad2302a2281e937d08R244 ?

Reply:
I have nothing against this function but as it was used at the end in a single place and the code is straightforward with object destructuring, I didn't think it was that useful.
But yeah, I agree it could come handy again in the future, so many we can keep it.

Reply:
I restored it :)